package.json

```diff
@@ -16,7 +16,7 @@
     "@logtrine/logtrine": "latest",
     "asar": "latest",
     "discord-rpc": "latest",
-    "electron": "8.3.4",
+    "electron": "^7.3.3",
```

Comment:
Should keep `8.3.4` if there is no compatibility issue with the new loading screen